lib/core/src/irods_parse_command_line_options.cpp

```diff
@@ -151,6 +151,7 @@ static int parse_program_options(
         _rods_args.progressFlag = 1;
     }
     if ( global_prog_ops_var_map.count( "rbudp" ) ) {
+        printf("Warning: RBUDP (-Q) is deprecated. Its use should be avoided.\n");
```

Comment:
Let's remove this since the help text states the option is deprecated.

Reply:
The rules have changed: https://github.com/irods/irods_docs/issues/124#issuecomment-2023055718

Reply:
I removed all warning messages.